src/bluesearch/database/topic.py

```diff
@@ -27,13 +27,13 @@
 
 
 # Journal Topic
-def request_mesh_from_nlm_ta(nlm_ta: str) -> list[dict] | None:
-    """Retrieve Medical Subject Heading from Journal's NLM Title Abbreviation.
+def request_mesh_from_journal_title(journal_title: str) -> list[dict] | None:
+    """Retrieve Medical Subject Heading from Journal's NLM Title.
```

Comment:
Then, is there at the end a difference between using the abbreviated title and the full title?

Reply:
By the way, why the change?

Reply:
(Just a side note, It is still a draft PR)
- Initially, when searching at `how to find the topic of an article`, I based the research on the `pmc` folder structure which is:
```
nlm_abbreviated_title/
    PMCXXX.nxml
    ....
```
So it was making sense to look at the journal thanks to the abbreviated title.
-  Thinking at a more generic strategy (we won't always parse PMC articles coming directly from the mentioned structure), I parsed the PMC files we have to see which info was communicated about the journal. The title seems more often specified than the journal abbreviated title. 
This is the reason for the change. But this is a draft PR, I am still considering staying with the search of abbreviated title or even have the two possibilities in the source code.